### DISCUSSION TITLE
お題のテーマ部分をコピー可能にする。

### DISCUSSION THREAD

**Discussion by haruharu-1105** [aipictors/app#123] [アイデア - Ideas]
## どのような問題がありますか？
お題のテーマで検索する時、テーマ名を入力するのが手間です。
（分からない漢字の時困ります。）

## どうすれば良いですか？
![image](https://github.com/aipictors/app/assets/132221228/962200c2-fdc1-421a-b29d-ef3edf15a407)

橙色囲み部分をタップで範囲選択 OR お題をコピーできるようにするのはどうでしょうか？
※動作イメージとしては、バージョン情報欄のタップと同じイメージになります。

**Comment by Stella2211** [comment]
とても便利だと思ったので、実装しました。

https://github.com/aipictors/app/assets/104188098/3123ce36-b419-4ed9-bef9-26c3f04c374a
